cmd/ketch/job_list_test.go

```diff
@@ -109,6 +110,24 @@ func TestJobListNames(t *testing.T) {
 			},
 			wantOut: []string{"hello"},
 		},
+		{
+			name: "filter",
```

Comment:
This test and the one below it share the same name, so it might be better to change the names to something like `filter - no match` and `filter-success`.